.github/ISSUE_TEMPLATE/request-new-sample.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Request New Sample
+about: Suggest an idea for a new sample project
```

Comment:
should we formulate somehow that this is not for new feature requests? (or is it? 🤔 )
So which existing feature should be showcased.

Reply:
No its not for SDK feature requests. We will update the README as well to provide more information around issues with the SDK rather than the samples. 

The use-case question is there to (maybe) derive future features from it.